### DISCUSSION TITLE
Remove null from ridbag

### DISCUSSION THREAD

**Discussion by johny65** [orientechnologies/orientdb#9980] [Q&A]
I have some vertices where a `null` is in their ridbags (I don't know why... see image). How can I remove those nulls? Because of them I'm having various `NullPointerException` from Java code.

![Captura desde 2023-05-10 11-18-10](https://github.com/orientechnologies/orientdb/assets/5256776/69e18ab5-a127-40cb-a427-17616aff2c1c)

**Comment by vRakhmail** [comment]
You can re-set the edge for vertex. You can do this on both vertexes sides by analogy:

update #238:393690
remove out_Domicilio_localidad

update #238:393690 
set out_Domicilio_localidad = [#*rid of the EDGE what was here*]

It helped me
